src/ports/postgres/modules/deep_learning/madlib_keras_fit_multiple_model.sql_in

```diff
@@ -1537,6 +1537,5 @@ CREATE AGGREGATE MADLIB_SCHEMA.fit_step_multiple_model(
     /* is_final_iteration */         BOOLEAN
 )(
     STYPE=BYTEA,
-    SFUNC=MADLIB_SCHEMA.fit_transition_multiple_model,
-    FINALFUNC=MADLIB_SCHEMA.fit_final_multiple_model
```

Comment:
It seems the sql and python functions for `fit_final_multiple_model` are still in code even though we don't call them anymore. Is there a particular reason for keeping them?

Reply:
No reason, I will remove them.